exchange/targeting_test.go

```diff
@@ -54,6 +56,11 @@ func TestTargetingCache(t *testing.T) {
 
 	assertKeyExists(t, bids["losing-bid"], string(openrtb_ext.HbCacheKey), false)
 	assertKeyExists(t, bids["losing-bid"], openrtb_ext.HbCacheKey.BidderKey(openrtb_ext.BidderAppnexus, maxKeyLength), false)
+
+	//assert hb_cache_host and hb_cache_path were included
+	assert.Equal(t, true, strings.Contains(string(bids["winning-bid"].Ext), string(openrtb_ext.HbConstantCacheHostKey)) && strings.Contains(string(bids["winning-bid"].Ext), "www.pbcserver.com"))
```

Comment:
The `assert` package has a `contains` [method](https://godoc.org/github.com/stretchr/testify/assert#Contains) that will do exactly what you want here and you don't have to assert over a bool value.

Reply:
Done. But have not updated the PR because I don't know if the `path` needs to be configurable as you mentioned in your second comment. @hhhjort